plugins/vst_base/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ IF(LMMS_SUPPORT_VST)
 INCLUDE(BuildPlugin)
 
 IF(LMMS_BUILD_WIN32)
-	ADD_DEFINITIONS(-DPTW32_STATIC_LIB)
+	ADD_DEFINITIONS(-DPTW32_STATIC_LIB -std=c++0x)
```

Comment:
> I guess -std=c++0x includes C++14 and some features of C++17.
Perhaps but that should be equally called `C++1y`, etc.  

https://herbsutter.com/2011/08/12/we-have-an-international-standard-c0x-is-unanimously-approved/